sample-templates/README.md

```diff
@@ -0,0 +1,33 @@
+## Flow Framework Sample Templates
+
+This folder contains sample workflow templates that can be used with Flow Framework.
+
```

Comment:
I know that we have some comments in the YAML template as you point below, but should we maybe have some info here just explaining the template more in depth or will this be in the documentation website?

For example as you also mention in YAML but to explain the edgs vs using previous nodes inputs.
Explaining that the type refers to whatever step and ID is set by user and can be anything, that user_input here corresponds to that API's payload.

It can still be a little overwhelming looking at the big template payload all together

Reply:
It's all explained in detail in the actual documentation on opensearch.org including a tutorial building it up piece by piece.  

https://github.com/opensearch-project/documentation-website/pull/6257/files/ba09447a5b3a48975a1759c4d16fa5b24d627bc0..8a1dca85a60a5cc6ce6d808c4cc6627c8b565636#diff-1007a4ccfce8e754061e7bf9a56298cf9a0832a39497bc09757c8b36a62292e9